src/main/scala/edu/berkeley/ce/rockslicing/JointSet.scala

```diff
@@ -0,0 +1,45 @@
+package edu.berkeley.ce.rockslicing
+
+/**
+  * Represents input joint set - allows for pattern matching to easily and clearly extract joint set fields
+  *
+  * @param parameters Array containing the input data representing the joint set. The inputs in each array
+  *                   are strike, dip, joint spacing, persistence, phi, cohesion and optional stochastic parameters
+  */
+case class JointSet(parameters: Array[Double]) {
```

Comment:
Remember that constructors are a little different for case classes. They determine how we can do pattern matching. So this really needs to be
```scala
case class JointSet(strike: Double, dip: Double, jointSpacing: Double, persistence: Double, phi: Double, cohesion: Double)
```
because it lets us do cool stuff like this:
```scala
jointSets map { case JointSet(strike, dip, jointSpacing, persistence, _, _) =>
    ...
}
```